code/datalab-app/src/app/components/modal/CreateNotebookDialog.js

```diff
@@ -1,32 +1,69 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import Dialog, { DialogTitle, DialogContent } from 'material-ui/Dialog';
+import dataStorageActions from '../../actions/dataStorageActions';
 import CreateNotebookForm from '../notebooks/CreateNotebookForm';
 import PreviewNotebookCard from '../notebooks/PreviewNotebookCard';
 
-const CreateNotebookDialog = ({ title, notebook, onSubmit, onCancel }) => (
-  <Dialog open={true} maxWidth="md">
-    <div style={{ margin: 10, display: 'flex', flexDirection: 'row' }}>
-      <div>
-        <DialogTitle>{title}</DialogTitle>
-        <DialogContent>
-          <CreateNotebookForm onSubmit={onSubmit} cancel={onCancel} />
-        </DialogContent>
-      </div>
-      <div style={{ width: 320 }}>
-        <DialogTitle>Notebook Preview</DialogTitle>
-        <div style={{ width: '90%', margin: '0 auto' }}>
-          <PreviewNotebookCard />
+class CreateNotebookDialog extends Component {
```

Comment:
This has gone from stateless component to a container. Should we have instead wrapped the stateless component? It would make the tests a lot easier as one set worry about the data and the other the presentation.